app/src/main/kotlin/com/google/samples/apps/nowinandroid/ui/interests2pane/InterestsListDetailScreen.kt

```diff
@@ -53,6 +54,8 @@ import java.util.UUID
 
 @Serializable internal object TopicPlaceholderRoute
```

Comment:
What's the reason that we have to `@Keep` the `DetailPaneNavHostRoute`, but not the `TopicPlaceholderRoute`?

Reply:
I'm not overly familiar with the minification process but my guess is that because the instance of `TopicPlaceholderRoute` is [referenced here](https://github.com/android/nowinandroid/blob/main/app/src/main/kotlin/com/google/samples/apps/nowinandroid/ui/interests2pane/InterestsListDetailScreen.kt#L98) it is not removed. 

On the other hand, `DetailPaneNavHostRoute` is only used as a parameterized type, never as an actual instance, so is removed.